.github/scripts/package_push.sh

```diff
@@ -19,9 +19,11 @@ docker build \
 
 if [[ "$GITHUB_REF" == 'refs/heads/master' ]]; then 
     docker push ${IMAGE_REPO}/${arch}_anax_debian:testing
-
     docker tag ${IMAGE_REPO}/${arch}_anax_debian:testing ${GITHUB_CONTAINER_REGISTRY}/${arch}_anax_debian:testing
     docker push ${GITHUB_CONTAINER_REGISTRY}/${arch}_anax_debian:testing
+else
+    # append the branch name to testing tags for when we're building older versions of anax for testing
+    docker push ${IMAGE_REPO}/${arch}_anax_debian:testing_${GH_BRANCH}
```

Comment:
The Docker build command produces the image with the `testing` tag. I think you need to tag it with `testing_${GH_BRANCH}` before pushing.